plugin-server/src/cdp/services/job-queue/job-queue-kafka.ts

```diff
@@ -90,6 +90,12 @@ export class CyclotronJobQueueKafka {
                             hogFunctionId: x.functionId,
                             functionId: x.functionId,
                             teamId: x.teamId.toString(),
+                            ...(x.queueScheduledAt && x.state?.returnTopic
```

Comment:
This is ugly and hard to read, can we just move the headers out to an object with an if statement?